B/Binaryen/build_tarballs.jl

```diff
@@ -0,0 +1,60 @@
+using BinaryBuilder, Pkg
+
+name = "Binaryen"
+version = v"1.0.0"
```

Comment:
Interesting question on version numbering. From what I can tell, Binaryen doesn't use semver and doesn't give any guarantees about compatibility. I guess I could follow their numbering as the lead digit. That'd mean this should be `v114.0.0`. If they did switch to semver, it'd screw up this system, though. So, maybe it'd be better to use `v0.114.0`. Then, each upgrade is still breaking. Either way, they release often enough that this might be a bit of a pain.

Reply:
Treating the version as a minor version for our use should be fine. A search in the binaryen repo actually shows they purposefully choose this scheme instead of semver (https://github.com/WebAssembly/binaryen/issues/2872, and https://github.com/WebAssembly/binaryen/issues/1156), and that they consider each release to be breaking actually (they say they can change API between releases). So from our perspective, having a compat for each version is probably the correct way forward (I guess it just means that the packages using binaryen will have to have updated compat entries whenever you do a new release).